src/trello/models.py

```diff
@@ -29,14 +30,14 @@ class Task(models.Model):
 	column = models.ForeignKey(Column, on_delete=models.CASCADE, null=False, related_name='tasks')
 	assignee = models.ForeignKey(User, on_delete=models.CASCADE, null=False, related_name='assigned_task')
 	title = models.CharField(verbose_name='Title', max_length=255)
-	due_date = models.DateField(verbose_name='Due date')
+	due_date = models.DateField(verbose_name='Due date', auto_now_add=True)
 	description = models.TextField(verbose_name='Description')
 
 
 class Subtask(models.Model):
 	task = models.ForeignKey(Task, on_delete=models.CASCADE, null=False, related_name='subtasks')
 	title = models.CharField(verbose_name='Title', max_length=255)
-	due_date = models.DateField(verbose_name='Due date')
+	due_date = models.DateField(verbose_name='Due date', auto_now_add=True)
```

Comment:
auto_now_add= True u can use only for fields like created_at, for due this is not work. Users should put this information